virtio-queue/Cargo.toml

```diff
@@ -13,16 +13,20 @@ edition = "2021"
 test-utils = []
 
 [dependencies]
-vm-memory = { workspace = true }
+vm-memory = {  workspace = true, features = ["backend-mmap", "backend-bitmap"] }
 vmm-sys-util = { workspace = true }
 log = "0.4.17"
 virtio-bindings = { path="../virtio-bindings", version = "0.2.5" }
+libc = "0.2.161"
```

Comment:
I'm a bit worried about new dependencies, why we now have to include libc and enable new backends for vm-memory under `[dependencies]` ?
(new deps under `[dev-dependencies]` are fine)

Reply:
refactored into new deps section only for `kani`